engines/bladerunner/bladerunner.cpp

```diff
@@ -1225,6 +1218,13 @@ void BladeRunnerEngine::handleKeyDown(Common::Event &event) {
 		return;
 	}
 
+	if (_vqaIsPlaying && !event.kbdRepeat && (event.kbd.keycode == Common::KEYCODE_ESCAPE || event.kbd.keycode == Common::KEYCODE_RETURN)) {
```

Comment:
It would be more preferable to leave this code in the keydown event handling, and add the kbdRepeat check there

Reply:
sorry, i haven seen this comment :(

Reply:
@bluegr I'm very confused. You're commenting in code that's in handlekeydown and saying "leave it in keydown"?